scripts/release/verify-fdroid-native-libs.sh

```
@@ -52,18 +186,59 @@ for apk in "${apks[@]}"; do
   extract_dir="$tmp_dir/$apk_name"
   mkdir -p "$extract_dir"
   unzip -qq "$apk" "lib/*/*.so" -d "$extract_dir"
+  patched_files=()
 
   while IFS= read -r so_file; do
-    if readelf -n "$so_file" | grep -q 'Build ID'; then
-      failures+=("$apk_name:${so_file#$extract_dir/}")
+    if has_build_id "$so_file"; then
+      rel_path="${so_file#$extract_dir/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)